import pytest

from invalid_argument_error import InvalidArgumentError

from x import X
def test_x_with_argument_raises_invalid_argument_error():
    with pytest.raises(InvalidArgumentError):
        X(1)